cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/type/DatasetTypeManager.java

```diff
@@ -96,6 +102,26 @@ protected void shutDown() throws Exception {
     // do nothing
   }
 
+  private Map<String, DatasetModule> getExtensionModules(CConfiguration cConf) {
+    Map<String, DatasetModule> modules = Maps.newLinkedHashMap();
```

Comment:
We use Java 7 syntax for creation of Map/Collection objects. `new LinkedHashMap<>();`